examples/ola-recorder.cpp

```diff
@@ -115,36 +121,51 @@ int VerifyShow(const string &filename) {
     return ola::EXIT_NOINPUT;
 
   map<unsigned int, unsigned int> frames_by_universe;
-  uint64_t total_time = 0;
 
-  unsigned int universe;
-  ola::DmxBuffer buffer;
-  unsigned int timeout;
+  ShowEntry entry;
   ShowLoader::State state;
+  unsigned int playback_pos = 0;
+  bool playing = false;
   while (true) {
-    state = loader.NextFrame(&universe, &buffer);
+    state = loader.NextEntry(&entry);
     if (state != ShowLoader::OK)
       break;
-    frames_by_universe[universe]++;
-
-    state = loader.NextTimeout(&timeout);
-    if (state != ShowLoader::OK)
+    playback_pos += entry.next_wait;
+    if (FLAGS_stop > 0 && playback_pos >= FLAGS_stop) {
+      // Compensate for overshooting the stop time
+      playback_pos -= (playback_pos - FLAGS_stop);
       break;
-    total_time += timeout;
+    }
+    if (!playing && playback_pos >= FLAGS_start) {
+      // Found the start point
+      playing = true;
+      entry.next_wait = playback_pos - FLAGS_start;
+    }
+    if (playing) {
+      frames_by_universe[entry.universe]++;
+    }
   }
 
+  const unsigned int total_time = playback_pos - FLAGS_start;
```

Comment:
Was there a reason for dropping this, and playback_pos from being uint64_t? That's less than a month in ms!

Reply:
I can definitely restore it - because all of the time-related flags are uint32_t (the largest type the flag macros allow), it seemed logical to use the same type.  My thought was one could either define a show with no time bounds that lasts until the heat death of the universe, or use bounds and have the show last up to about a month.  No strong feelings either way.

Reply:
Answering your comments in reverse:

> or use bounds and have the show last up to about a month.

I think that's the key, if it was over a month I wouldn't have said anything (I don't realistically expect a show to run for over a month), but monthly showfiles and a Christmas special don't seem too unlikely for example.

> I can definitely restore it - because all of the time-related flags are uint32_t (the largest type the flag macros allow), it seemed logical to use the same type.

I think in reality that suggests we possibly need an additional flag type, but that's probably one for another PR!

Reply:
The latest commit replaces time variables with a typedef for uint64_t.